editor/editor_themes.cpp

```diff
@@ -395,6 +419,7 @@ Ref<Theme> create_editor_theme(const Ref<Theme> p_theme) {
 	Color base_color = EDITOR_GET("interface/theme/base_color");
 	float contrast = EDITOR_GET("interface/theme/contrast");
 	bool increase_scrollbar_touch_area = EDITOR_GET("interface/touchscreen/increase_scrollbar_touch_area");
+	bool editor_icon_scale = get_editor_icon_scale();
```

Comment:
Should store the setting itself as a constant, so it's less arbitrary.

Reply:
@YuriSizov Do you mean to create a constant like `EDSCALE` for the gizmo handles scale?

Reply:
No, I mean for this bit of code where we store old values and compare them to learn if we need to regenerate the theme you can store the editor setting directly as a constant, instead of calling `get_editor_icon_scale()` without parameters.